flycheck.el

```diff
@@ -6678,7 +6678,7 @@ See URL `https://golang.org/cmd/go/' and URL
                 :face (if have-vet 'success '(bold error)))))))
 
 (flycheck-def-option-var flycheck-go-build-install-deps nil go-build
```

Comment:
Since the option now covers `go-test` as well you'll need to turn `go-build` into `(go-build go-test)`.